docs/kubernetes-installers.md

```diff
@@ -24,6 +24,7 @@ work with that Antrea version.
 | - | AKS, K8s v1.19.9 | Azure | Ubuntu 18.04, containerd | Standard_DS2_v2 |  | Antrea CI |
 | - | Kind v0.9.0, K8s v1.19.1 | N/A | Ubuntu 20.10, containerd://1.4.0 | N/A |  | [Requirements for using Antrea on Kind](kind.md) |
 | - | Minikube v1.25.0 | N/A | Ubuntu 20.04.2 LTS (5.10.76-linuxkit) arm64, docker://20.10.12 | 8GB RAM | | |
+| v1.11.0 | Kubeadm v1.20.2 | N/A | openEuler 22.03 LTS, docker://18.09.0 | 10GB RAM | | |
```

Comment:
Kubernetes v1.20.2 is pretty old and releases older than 1.24 are no longer maintainerd, can you verify it with a more recent version? Otherwise the information wouldn't be very helpful for reference.

Reply:
I chose to test Kubernetes v1.20.2 because it is the latest version officially supported by openEuler 22.03. http://repo.openeuler.org/openEuler-22.03-LTS/update/x86_64/Packages/.

I can manually upgrade K8s and do the verification under more recent versions.

Reply:
OK, then let's also keep this one if this is what users will get by default.

Reply:
updated